custom-words.txt

```diff
@@ -2785,4 +2785,8 @@ Unprepares
 Unpreparing
 typespec
 DSPM
-Dspm
\ No newline at end of file
+Dspm
+Gtid
+GTID
+gtid
+Dspm
```

Comment:
two Dspm?

And where are them used?

Reply:
this is caused my resolving conflicts. Let me fix